Procfile

```diff
@@ -1 +1 @@
-web: node lib/main start | ./node_modules/.bin/bunyan -o short
```

Comment:
Sorry, I'd like to leave this as `lib/`.  This tool was the first to use babel-less node again, and has used `lib/` from day one.  Best practises should have been written after examining prior art like this repository.

As for the reason, this code is not source for a compiler, rather it's executable code.

Reply:
I find these little differences trip me up repeatedly when working across multiple services, for example when upgrading a widely-used TC library.  Remembering that ec2-manager is "different" is just a little cognitive drag.

It will take quite a bit of time to shift every other service over to `lib/`, rather than shifting this one to `src/`.  Is it worth spending that time for three letters?

Reply:
We agreed to just leave these different.